graphql_compiler/compiler/ir_lowering_common/location_renaming.py

```diff
@@ -1,11 +1,17 @@
 # Copyright 2019-present Kensho Technologies, LLC.
 """Utilities for rewriting IR to replace one set of locations with another."""
+from typing import Callable, Dict
+
 import six
 
-from ..helpers import FoldScopeLocation, Location
+from ...compiler.expressions import ExpressionT
+from ..helpers import FoldScopeLocation, Location, LocationT
+from ..metadata import QueryMetadataTable
 
 
-def make_revisit_location_translations(query_metadata_table):
+def make_revisit_location_translations(
+    query_metadata_table: QueryMetadataTable,
+) -> Dict[LocationT, LocationT]:
```

Comment:
```suggestion
) -> Dict[Location, Location]:
```
This is not apparent here, but will become apparent once that `from ..metadata` import up there is typed.